### PR TITLE
Add results to point search panel

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/pages/MapPage/Components/PointSearchResults.tsx [hunk 1]
```diff
@@ -1,49 +1,152 @@
...
-          quam quibusdam sapiente similique sit suscipit ut vel vero! Accusamus ad consequatur
-          dolore esse, facere fugiat illum maxime mollitia nihil optio, quasi quisquam reprehenderit
-          saepe sunt totam unde vel veniam veritatis vero voluptates.
-        </Paragraph>
-      )}
-    </MapPanelContent>
+      <CoordinatesText>
+        <strong>Locatie:</strong> {location.lat}, {location.lng}
+      </CoordinatesText>
+      <PanoramaPreview location={location} radius={180} aspect={2.5} />
```
Comment (royderks) [inline src/app/pages/MapPage/Components/PointSearchResults.tsx:99]: Should we define the radius and aspect somewhere? Or have defaults for the panoramapreview component?

File: src/app/pages/MapPage/Components/PointSearchResults.tsx [hunk 2]
```diff
@@ -1,49 +1,152 @@
...
+      ))}
+    </ShowMore>
+  )
+}
+
+const mapStateToProps = (state: RootState) => ({
+  user: getUser(state),
+})
+
+export default connect(mapStateToProps)(PointSearchResults)
```
Comment (royderks) [inline src/app/pages/MapPage/Components/PointSearchResults.tsx:152]: Do we still want to use the HOC for redux here, or already decouple a bit more and use the `useSelector` hook from react-redux? Also, there is a getState function in the codebase to get the redux code with a HOC

File: src/app/pages/MapPage/Components/PointSearchResults.tsx (hunk 1 above)
Comment (jonkoops) [reply src/app/pages/MapPage/Components/PointSearchResults.tsx:99]: I think it's fine to keep these in-line here, the API itself already provides defaults so I'd rather not override these.

File: src/app/pages/MapPage/Components/PointSearchResults.tsx (hunk 2 above)
Comment (jonkoops) [reply src/app/pages/MapPage/Components/PointSearchResults.tsx:152]: I wasn't aware of this hook, but it looks much better. I've changed the code to use it instead.

**Pull request merged**